contract/models/account_analytic_account.py

```diff
@@ -236,10 +236,30 @@ def _prepare_invoice(self):
         return invoice._convert_to_write(invoice._cache)
 
     @api.multi
-    def _create_invoice(self):
+    def _prepare_invoice_update(self, invoice):
+        vals = self._prepare_invoice()
```

Comment:
This is still overwriting all the previous values, where they should be more important than new ones.

Reply:
OK, got it